cbits/bctable.h

```diff
@@ -0,0 +1,32 @@
+#include <Rts.h>
+#include <stdlib.h>
+
+/// A type of bytecodes
+struct inline_java_dot_class {
+    char* ij_dc_class_name;
+    int ij_dc_bc_count;
+    unsigned char* ij_dc_class_bytecode;
+};
+
+/// A type of linked lists for bytecode
+///
+/// NULL is the empty linked list.
+struct inline_java_linked_list
+  { int ij_ll_dc_count;
```

Comment:
The documentation for `dc_count` doesn't tell me what your invariant is. Does the `dc_count` field always have the same value at every node in the list, or does it cache the length of the remaining items in the list?